scripts/react_native_pods.rb

```diff
@@ -33,6 +33,8 @@ def use_react_native! (options={})
   # Include Hermes dependencies
   hermes_enabled = options[:hermes_enabled] ||= false
 
+  flipper_configuration = options[:flipper_configuration] ||= FlipperConfiguation.disabled
```

Comment:
I think that the default should be `.enabled`. Previously the `use_flipper!` function was always invoked.

Reply:
It was a separated function, so I think it was up to the final user to call it or not, did I miss something?

Reply:
Yes, but it was always invoked, if the `use_framework` was `false`, therefore, the default should be truthy (or to install the dependencies). Does it make sense?

Reply:
Ok, you are right. My suggestion was there because:
1. In RNTester, `use_flipper!` is always invoked.
2. In the [Template](https://github.com/facebook/react-native/blob/main/template/ios/Podfile#L32) `use_flipper!` is there by default.

And I guess that a vast majority of projects uses the template _as-is_.

Therefore, I'd change the suggestion to... could you update the template as well? 😊